HelloMinimal-CMake/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+# This is the most minimal version of a CMake file to compile an executable and
+# a library. Refer to HelloWorld-CMake for a more fleshed out example project.
```

Comment:
I think that this belongs in a readme instead.  Could you please add that to a readme markdown file in the directory?